tasks/LaundrySort.tex

```diff
@@ -0,0 +1,44 @@
+\section{Laundry Sort [Housekeeper]}
+The robot is helping to do laundry by sorting it.
+
+% \subsection{Focus}
+% This test focuses on object perception, manipulation, and planning.
+
+\subsection{Main Goal}
+The robot is given a bin of laundry and must sort it by color and size. There are both children's and adult's clothing in the bin, and the robot must make piles by color and size. There are existing piles of laundry on the floor into which the laundry in the bin must be sorted.
```

Comment:
Handling clothes is an entire problem on its own, yet I see here 3 features:
- Grasping from containers
- Grasping from the floor
- Cloth manipulation
- Size classification (this might require 2-handed manipulation)
- Color classification

Too many features to address, even for a Stage II test. I would agree if weren't because at least 3 have never been addressed in @Home.

The goal needs some rework:

> The sorts clothing by color and size, separating them by color [and size] in heaps on the table/bin.

For some reason *bin* makes me thing on garbage. My problem I guess